plugins/importexport/native/filter/SubmissionNativeXmlFilter.php

```diff
@@ -196,7 +197,7 @@ public function addPublications($doc, $submissionNode, $submission)
                 $submissionNode->appendChild($clone);
             } else {
                 $deployment = $this->getDeployment();
-                $deployment->addError(ASSOC_TYPE_SUBMISSION, $submission->getId(), __('plugins.importexport.publication.exportFailed'));
+                $deployment->addError(PKPApplication::ASSOC_TYPE_SUBMISSION, $submission->getId(), __('plugins.importexport.publication.exportFailed'));
```

Comment:
Here, again, maybe we should pick a consistent way to deal with that `$deployment = $this->getDeployment();`. We could do that either `$deployment = $this->getDeployment();` and use the `$deployment` after or by using `$this->getDeployment()->`.

Reply:
Yep, in general I just create variables if I'm going to use them at least twice. The codebase has both patterns 🤔